secantTests/AppTests/AppTests.swift

```diff
@@ -88,43 +88,76 @@ class AppTests: XCTestCase {
     }
 
     func testRespondToWalletInitializationState_FilesMissing() throws {
+        let testScheduler = DispatchQueue.test
+
         let store = TestStore(
             initialState: .placeholder,
             reducer: AppReducer()
         )
```

Comment:
All changes here in the whole file look unnecessary. When I tried the previous code, it works.

Reply:
Taken back.